### ISSUE TITLE
Is it easy to support variable weight using point size?

### ISSUE THREAD

**Issue by ShixiangWang** [NikNakk/forestmodel#28]
In the model, weight may contain important information. However, in current forestmodel, the point size is fixed.
So I wonder it would be nice to implement this.

https://github.com/NikNakk/forestmodel/blob/dd8eb5579194ce13d78f6539f9e1c48b69c4b5d8/R/panel_forest_plot.R#L366-L370

**Comment by NikNakk** [comment]
How would you propose scaling the points? I understand for a meta-analysis forest plot, but not one for a `glm`.

**Comment by ShixiangWang** [comment]
@NikNakk Yeah, this is true. Some models may find it useful, but some others may not.

**Comment by NikNakk** [comment]
Happy to review suggested code (or even just a suggested algorithm for calculating point size).

**Comment by ShixiangWang** [comment]
Thanks, I will look into it in my spare time.
